extensions/wikia/EditPageLayout/js/plugins/WikitextSyntaxHighlighter.js

```diff
@@ -207,20 +207,25 @@ define('WikiTextSyntaxHighlighter', ['wikia.window', 'wikia.document'], function
 							//some other kind of tag, search for its end
 							//the search is made easier because XML attributes may not contain the character ">"
 							var tagEnd = text.indexOf(">", i) + 1;
+							/**
+							 * Fandom change
+							 * - variable was moved here from nested `else` block
+							 * - variable is used in `if` statement below
+							 */
+							var tagName = match[0].substring(1);
+
 							if (tagEnd === 0) {
 								//not a tag, just a "<" with some text after it
 								writeText("<", color);
 								i = i - match[0].length + 1;
 								break;
 							}
 
-							if (text.charAt(tagEnd - 2) === "/") {
+							if (text.charAt(tagEnd - 2) === "/" || tagName === "br") {
```

Comment:
Hmm, this looks pretty exclusive to just BR tags right now. Are there any more tags we should be watching out for? Like e.g. hr? What about other self-closing tags?

Reply:
This change was originally introduced 6 years ago as stated [here](https://github.com/Wikia/app/blame/048c536083c05e0818bbbb130b9a616026cc2260/extensions/wikia/EditPageLayout/js/plugins/WikitextSyntaxHighlighter.js#L294) and lasted unchanged till a recent update so I'd expect any additional tags to be there by now. 

I've checked it for `<hr/>` and `<hr>` and both are highlighted.